.env

```diff
@@ -55,6 +57,9 @@ DISTRIBUTOR_QUERY_NODE_URL=http://graphql-server:8081/graphql
 # Indexer gateway used by processor. If you don't use the local indexer set this to a remote gateway
 PROCESSOR_INDEXER_GATEWAY=http://hydra-indexer-gateway:4000/graphql
 
+# State update endpoint used by prcessor (to send state updates)
+STATE_UPDATE_ENDPOINT=http://graphql-server:8082/update-processor-state
```

Comment:
Doesn't this need to be passed in env for the processor? (As in the e2e test setup in hydra repo)

Reply:
the default using 'localhost' probably doesn't work on mac with docker desktop..

```
    STATE_UPDATE_ENDPOINT: str({
      devDefault: 'http://localhost:8082/update-processor-state',
    }),
```

Reply:
> Or is it being passed through .env file through use of the env_file directive ?

Yes, it should be passed because of this directive, it doesn't need to be explicitly named in `environment`

> the default using 'localhost' probably doesn't work on mac with docker desktop..

I'm aware it doesn't work in some setups, but it's just the default. I followed the approach already used for indexer endpoint, ie.: `INDEXER_ENDPOINT_URL: str({ devDefault: 'http://localhost:4001' }),`